recipe/meta.yaml

```diff
@@ -1,66 +1,46 @@
-{% set version = "0.5.11" %}
 {% set org = "pyne" %}
-{% set sha256 = "ba05917b0de446ec520857655e77967a783e3bc723299faf5989f532d6b0647a" %}
-{% set variant = "openblas" %}
+{% set version = "0.7.0" %}
+{% set build = "0" %}
+{% set sha256 = "19e104a05aa56ef2e747f5e4a1c4cffefdb57a592a300823265ba72a1b55d4f3" %}
 
 package:
   name: pyne
   version: {{ version }}
 
 source:
   fn: pyne-{{ version }}-{{ sha256 }}.tar.gz
-  url: https://github.com/{{ org }}/pyne/archive/{{ version }}.tar.gz
+  url: https://github.com/{{ org }}/pyne/archive/{{ version }}-rc1.tar.gz
```

Comment:
Please see the very new instructions for releasing rcs: https://github.com/conda-forge/cfep/blob/master/cfep-05.md

Reply:
Thanks @scopatz - we'll review this.  Part of our reason for issuing an RC was to allow us to dig into the conda-forge workflow without waiting to figure out a PyNE release.  We are hoping/imagining that a release may replace this RC soon.

Nevertheless, we'll look at the docs.